containers/src/impl/Kokkos_UnorderedMap_impl.hpp

```diff
@@ -223,16 +222,14 @@ struct UnorderedMapPrint {
 
   map_type m_map;
 
-  UnorderedMapPrint(map_type const& map) : m_map(map) {}
```

Comment:
Why did you change?

Reply:
For the same reason as for `UnorderedMapErasure`, see my comment https://github.com/kokkos/kokkos/pull/6610#discussion_r1402100981 #5980